src/Controllers/BatchProcessorController.ts

```diff
@@ -415,19 +431,59 @@ module StreamStats.Controllers {
                     }
 
                     else {
-                        // this.streamStatsAvailable = false;
-                        // this.toaster.pop('error', "No parameters available for this region", "Please select another region");
                     }
                     return paramRaw;
                 }, (error) => {
-                    // console.log('Bad response from the regression service');
-                    // this.streamStatsAvailable = false;
-                    // this.toaster.pop('warning', "StreamStats not available here at this time", "", 5000);
-
                 }).finally(() => {
                 });
         }
 
+        public submitBatch(): void {
+
+            // validate that the file is a .zip
+            if (this.submitBatchData.attachment.type != "application/zip") {
+
+                this.toaster.pop('warning', "Please select a .zip file to upload and click Submit Batch again.", "", 5000);
+                return;
+            }

```

Comment:
Validate the zip file in a separate function

```suggestion
        public validateZipFile($files): void {
            // validate that the file is a .zip
            if ($files[0].type != "application/zip") {
                this.toaster.pop('warning', "Please upload a .zip file.", "", 5000);
                this.submitBatchData.attachment = null;
            }
            return;
        }
        
        public submitBatch(): void {

```